scripts/server.mjs

```diff
@@ -0,0 +1,10 @@
+import { createServer } from "http-server";
+
+const server = createServer({ root: process.env.SERVER_DIR });
+server.listen(process.env.SERVER_PORT);
+server.server.on('listening', () => {
+    process.send?.({port: server.server.address().port});
+})
+server.server.on('error', () => {
+    process.exit(1)
+})
```

Comment:
this script just allows us to programmatically create a server on the fly (using the same library we already use) - this is used in the fingerprinting tests to start the second server